index.html

```diff
@@ -182,15 +182,15 @@ <h3>
       provider=].
     </p>
     <h2>
-      The `IdentityRequestProvider` dictionary
+      The `IdentityRequestDetails` dictionary
```

Comment:
```suggestion
      The `PresentationRequest` dictionary
```

A request for a digital movie ticket is not a request for a digital identity... it's a request for the presentation of a digital credential.

Terminology that seems to have more consensus than "digital identity request" is the concept of a "presentation" (this is most likely  across mDL, mDoc, VCs, AnonCreds, and other emerging digital credential formats).  

Has this terminology been considered before?

https://w3c.github.io/vc-data-model/#dfn-presentation

Reply:
As the dictionary name is not exposed anywhere, it's only really helpful on the C++ side (and potentially via the browser's dev tools) - so generally what we want is something pretty specific here, hence the updated `DigitalCredentialsProvider` name.